portal-api/src/v1/users/sign-in-link.controller.js

```diff
@@ -1,10 +1,45 @@
+const { LOGIN_STATUSES } = require('../../constants');
+const utils = require('../../crypto/utils');
+const { updateLastLogin } = require('./controller');
+const { sanitizeUser } = require('./sanitizeUserData');
+
 class SignInLinkController {
   #signInLinkService;
 
   constructor(signInLinkService) {
     this.#signInLinkService = signInLinkService;
   }
 
+  async validateSignInLink(req, res) {
+    try {
+      const { signInToken } = req.params;
+      const { user } = req;
```

Comment:
```suggestion
      const { params: { signInToken }, user } = req;
```
^ A minor suggestion, I don't know if you have a preference

Reply:
I have no strong preference either way, updated